### PR TITLE
Color fix

### PR DESCRIPTION
**Pull request by shvelo**
Fix crash when color index is invalid.

I couldn't create a test case for this because I don't know how to edit DXF files and couldn't find the file with invalid colors.

### CODE REVIEW

File: lib/toSVG.js [hunk 1]
```diff
@@ -52,7 +53,12 @@ module.exports = function(parsed) {
     if (!layerTable) {
       throw new Error('no layer table for layer:' + entity.layer);
     }
-    const rgb = colors[layerTable.colorNumber];
+
+    let rgb = colors[layerTable.colorNumber];
+    if(rgb == null) {
```
Comment (bjnortier) [inline lib/toSVG.js]: 1. should be ```if (``` to confirm to the rest of the codebase
1. should be ```===``` not ```==```, which requires ```=== undefined```

Comment (bjnortier) [inline lib/toSVG.js]: Pls use === so I can merge this

**Comment by shvelo** [comment]
You're right, out of bounds array access is always `undefined`, no point in `== null`

**Comment by bjnortier** [comment]
Thanks! Waiting for travis then I'll merge it

**Comment by bjnortier** [comment]
```
> undefined == null
true
```
so it would have been fine, but I never use ``==``

**Pull request merged**